ide/_tags

```diff
@@ -0,0 +1,16 @@
+# To build:
+#
+# - ocamlbuild -use-ocamlfind idetop.native && mv idetop.native coqidetop
+# - cp coqide_main.ml{4,} && ocamlbuild -use-ocamlfind coqide_main.native && mv coqide_main.native coqide
+
+true: rectypes, thread, package(dynlink,coq.lib)
+
+<protocol>: include
+
+# Language server
+<idetop.*>: package(coq.toplevel)
+<idetop.native>: linkall
+
+# CoqIDE itself
+not <idetop.ml>: package(lablgtk2.sourceview2)
+<coqide_main.ml>:  syntax(camlp4o), package(camlp4.macro)
```

Comment:
Why camlp4 and not camlp5?

Reply:
That's a good question, it likely comes from the time the legacy repo was created.